todo_app/pymongo_practice.py

```diff
@@ -0,0 +1,14 @@
+import pymongo
+from  datetime import datetime
+from bson import ObjectId
+
+id = "612dfa5e366842b65f9c9d33"
+client = pymongo.MongoClient(
+    "mongodb+srv://admin:MongoAdmin1@cluster0.qtpde.mongodb.net/myFirstDatabase?retryWrites=true&w=majority", ssl=True, ssl_cert_reqs='CERT_NONE')
```

Comment:
This line still has your mongo password in, Ben has already advised rotating the password so I'll trust you have, but we should still remove the old one from here (and if you haven't rotated it, do!)

Reply:
Removed, thank you